### PR TITLE
Bump to slirp4netns v0.4.3

### PR DESCRIPTION
**Pull request by ibuildthecloud**

### CODE REVIEW

File: slirp4netns/slirp4netns.mk [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-SLIRP4NETNS_VERSION = 6a2bb52fa6d888dc355344c2c45c8a80d1b41c3a
+SLIRP4NETNS_VERSION = v0.4.3
 SLIRP4NETNS_SITE = git://github.com/rootless-containers/slirp4netns.git
 SLIRP4NETNS_LICENSE = BSD-4-Clause, BSD-2-Clause
```
Comment (AkihiroSuda) [inline slirp4netns/slirp4netns.mk]: GPL-2.0-or-later

**Comment by AkihiroSuda** [comment]
ping

**Comment by AkihiroSuda** [comment]
v1.0.1 is ready now

**Comment by erikwilson** [comment]
Thanks for the info @AkihiroSuda! I am trying to create a release which includes the new version of slirp4netns but have been having lots of problems.

Unfortunately it looks like the new glibc dependency could be problematic, also trying to figure out what needs to be done for libcap or libseccomp. 
Additionally the version check for slirp doesn't seem to line up with what is provides by buildroot, was having a hard time tracking down that specific version requirement.

**Comment by AkihiroSuda** [comment]
I suggest compiling slirp4netns statically in Dockerfile without buildroot

**Comment by erikwilson** [comment]
Thanks for the info @AkihiroSuda, I think that is the best approach also as it appears a lot will need to be changed for buildroot.

In that case it may be best for us to setup another repo for building just slirp4netns, but if you are able to publish static binary releases for amd64, arm64, and arm, that might be nice as we can just download those releases in the k3s repo.

**Comment by AkihiroSuda** [comment]
> if you are able to publish static binary releases for amd64, arm64, and arm, that might be nice as we can just download those releases in the k3s repo.

Available now: https://github.com/rootless-containers/slirp4netns/releases/tag/v1.1.0-beta.1

**Comment by AkihiroSuda** [comment]
v1.1.0 is GA now: https://github.com/rootless-containers/slirp4netns/releases/tag/v1.1.0

**Comment by ibuildthecloud** [comment]
@AkihiroSuda thanks!  I'll try to update this to pull in the static binary instead of us building it. We are switching the CI around for k3s-root so might take a week or so.  But thanks for pushing the static binaries.

**Comment by brandond** [comment]
Do we want to close this out in favor of #24?

**Comment by brandond** [comment]
Closing since #24 was merged

**Pull request closed without merging**